### ISSUE TITLE
MAME 2003 Plus core

### ISSUE THREAD

**Issue by filevans** [SuperrSonic/RA-SS#43]
hello, please could you add **MAME 2003 Plus**core and **Final Burn Neo** core? as they have better compatibility for arcade games

thank you very much for listening

**Comment by SuperrSonic** [comment]
Neo is too big, it won't run unless you manually pick which drivers to build, which is what I did, the latest release contains a CPS-1/CPS-2 core, it's not clear which drivers would be beneficial for Wii, so I just built these two for better compatibility, maybe.

This other mame core, I feel I've released it before, it's possible I omitted it in the last release, due to obscurity and lack of info.

**Comment by filevans** [comment]
So has MAME 2003 Plus core been included or not? As mame 2000 has bad compatibility
And regarding FB neo, the CPS 1 and 2 and neo geo from it weren't required as compatibility was already perfect for those 3 drivers on FBA

One game that was not compatible with FBA was altered beast and the speech played high pitched with some other sound glitches. FB neo plays it perfectly. I think that's Sega system 16 driver, so if you could just build FB neo with the sega sys 16 driver that would be helpful

**Comment by filevans** [comment]
Please could you just include MAME 2003 or MAME 2003 core? as they have much better compatibility than MAME 2000, also could you build just the Sega driver from FB Neo which would cover virtually all the main stuff
thanks

**Comment by SuperrSonic** [comment]
I've just added back the FBneo build with sega drivers. I tested Cotton US ver. the title is a bit slow but gameplay seems fine.
I could not find a mame 2003 in my files that are not the split cores by Wiimpathy, which are already included.

It's been a while, but there actually was a good reason to include FBNeo with cps-1, they added some interesting hacks for one of the Final Fight games and including a modern cps-2 could result in better handling of things like savestate creation.
